polkadot-parachains/pal-parachain/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
-name = "rococo-parachain-runtime"
+name = "pal-parachain-runtime"
```

Comment:
What does pal stands for? 

I don't get the renaming?

Reply:
(sorry I thought I replied to this via email but must not have sent)
Pal is another word for friend. In order to test parachains xcm functionality you need other parachains to talk to. In particular we'd like this chain to mint and burn derivative-tokens to represent the reserve asset being transferred as this would simulate how we expect others to interact with statemine/t and also be an example to others of how such can be supported.

(we could keep rococo-parachain and have Pal as a separate runtime if people still have need of it.)

Reply:
I mean we can change it, but can we not come up with a different name? AFAIK it is also still being used for Versi testing. 

> example to others of how such can be supported.

You are speaking again about tutorials. We should create a tutorial for this and not really have some code lying around that no one understands.